### PR TITLE
Added recent Windows 10 SDK versions to cmake qt files

### PR DESCRIPTION
**Pull request by glenn-saic**
Not sure if this can be better achieved with a wildcard.

Change also applies to same file in SMDIS plugin-api

### CODE REVIEW

**Comment by emminizer** [comment]
Thanks, this looks like a good change. I will try to get this merged in later today. I agree the definitions are tedious.

**Comment by glenn-saic** [comment]
I commented out the section and cmake seemed to generate the plugin-api without issue. Should be fine.

 

Glenn

 

 

From: Daniel Emminizer ***@***.***> 
Sent: Monday, April 10, 2023 12:19 PM
To: USNavalResearchLaboratory/simdissdk ***@***.***>
Cc: Kinnear, Glenn T. [US] ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [USNavalResearchLaboratory/simdissdk] Added recent Windows 10 SDK versions to cmake qt files (PR #94)

 

EXTERNAL EMAIL -- This message originates from outside of SAIC

Hi @glenn-saic <https://github.com/glenn-saic>  -- I am looking at this again and having a hard time triggering an issue on my systems. I am considering removing this whole block that searches for glu32.lib and updates the CMAKE_PREFIX_PATH. Do you have a system where such a change would break your build?

Basically the entire if statement here would go away, from if(WIN32 AND on 51 to its endif() on 90.
